README.md

```diff
@@ -5,33 +5,82 @@
   <img src="./.github/assets/topos_logo_dark.png#gh-dark-mode-only" alt="Logo" width="200">
   <br />
   <p align="center">
-  <b>Topos Playground</b> is a CLI make it simple to run a local Topos devnet 🚀
+  <b>Topos Playground</b> is a CLI to make it simple to run a local Topos devnet 🚀
   </p>
   <br />
 </div>
 
 ## Getting Started
 
+The Topos Playground is a CLI that allows you to easily run and manage a local Topos devnet. It depends on Docker and NodeJS, and manages a number of components in order to locally run multiple subnets, multiple TCE nodes, and a dApp with supporting infrastructure, capable of cross-subnet messaging.
+
 ### Requirements
 
+- [Docker](https://docs.docker.com/get-docker/_) version 17.06.0 or greater
+- [NodeJS](https://nodejs.dev/en/) version 16.0.0 or greater
 - Git
-- NodeJS (tested with 16+)
-- Docker
 
-### [Optional] Install the package globally
+### Install / Run topos-playground
+
+Depending on your NodeJS environment and preferences, there are several ways to install the Topos Playground.
```

Comment:
Nitpick: I think we should suggest a "best" way here, at least implicitly by having it as the first option. Is a global  install really our preference (`npm i -g`)?

Reply:
I think that it is, as otherwise one has to be in the topos-playground repo to run it, instead of it being a command that one can invoke from any directory.

Reply:
Fair. I have no horse in that particular race, but I know some JS devs that are very peculiar about global installs. I guess such people will want to clone the repo anyway.